students/saleta_sebastian/lesson_06_dicts_tuples_sets_args_kwargs/fantasy_game_inventory.py

```diff
@@ -0,0 +1,28 @@
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+inv = {'gold coin': 42, 'rope': 1}
+dragonLoot = ['gold coin', 'dagger', 'gold coin', 'gold coin', 'ruby']
+
+
+def displayInventory(inventory):
```

Comment:
According to PEP8 function name should be lowercase separated by an underscore 😄